src/priorityq.py

```diff
@@ -0,0 +1,55 @@
+"""This module contains an implementation of the PriorityQ."""
```

Comment:
More explanation about how to use this module would be nice here...how priority is scored, etc...